### PR TITLE
Add copy and combine method to running mean std

### PR DESCRIPTION
**Pull request by araffin**
## Description

Needed for https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/pull/42

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: tests/test_vec_normalize.py [hunk 1]
```diff
@@ -202,6 +202,33 @@ def test_runningmeanstd():
         assert np.allclose(moments_1, moments_2)
 
 
+def test_combining_stats():
+    np.random.seed(4)
+    for shape in [(1,), (3,), (3, 4)]:
+        values = []
```
Comment (Miffyli) [inline tests/test_vec_normalize.py:208]: Is `values` needed?

File: tests/test_vec_normalize.py [hunk 2]
```diff
@@ -202,6 +202,33 @@ def test_runningmeanstd():
...
+        rms_1 = RunningMeanStd(shape=shape)
+        rms_2 = RunningMeanStd(shape=shape)
+        rms_3 = RunningMeanStd(shape=shape)
+        for _ in range(15):
+            value = np.random.randn(*shape)
+            rms_1.update(value)
+            rms_3.update(value)
+            values.append(value)
+        for _ in range(19):
+            rms_2.update(value)
```
Comment (Miffyli) [inline tests/test_vec_normalize.py:218]: More nitpick, but I assume here we just want to skew the stats towards `value` for a test (away from normal distribution)?

**Review by Miffyli — COMMENTED** [review]
LGTM, just few minor comments regarding tests :)

File: tests/test_vec_normalize.py (hunk 2 above)
Comment (araffin) [reply tests/test_vec_normalize.py:219]: ah no, just me copy pasting code without thinking too much :see_no_evil:

**Review by Miffyli — APPROVED** [review]

**Pull request merged**